e2e/ctl_v3_auth_test.go

```diff
@@ -454,3 +459,95 @@ func authTestPrefixPerm(cx ctlCtx) {
 		cx.t.Fatal(err)
 	}
 }
+
+func authTestMemberAdd(cx ctlCtx) {
+	if err := authEnable(cx); err != nil {
+		cx.t.Fatal(err)
+	}
+
+	cx.user, cx.pass = "root", "root"
+	authSetupTestUser(cx)
+
+	// ordinal user cannot add a new member
+	cx.user, cx.pass = "test-user", "pass"
+	peerURL := fmt.Sprintf("http://localhost:%d", etcdProcessBasePort+11)
```

Comment:
OK, I'll add new fledged functions and modify callers in both of `ctl_v3_auth_test.go` and `ctl_v3_member_test.go`.